public/app/percona/check/Check.service.ts

```diff
@@ -34,12 +38,19 @@ export const CheckService = {
       token
     );
 
-    return result.map(({ service_name, service_id, critical_count = 0, warning_count = 0, notice_count = 0 }) => ({
+    return result.map(({ service_name, service_id, ...counts }) => ({
```

Comment:
Is there a reason why not to destruct counts as well?

Reply:
good one 👍